deployment/nginx/uwsgi.conf

```diff
@@ -17,12 +17,13 @@ server {
         uwsgi_param REMOTE_ADDR $http_x_forwarded_for if_not_empty;
         uwsgi_param REMOTE_USER $http_x_userid if_not_empty;
         uwsgi_param GEN3_REQUEST_TIMESTAMP $msec;
-        uwsgi_param GEN3_TIMEOUT_SECONDS '45';
+        uwsgi_param GEN3_TIMEOUT_SECONDS $arg_timeout;
```

Comment:
where is this set?

Reply:
oh ok

Reply:
:gratitude: Yes, thanks! Just for the record, it is set in Kubernetes configuration in the URL `/_status?timeout=2`.